src/my-workspace/constants.ts

```diff
@@ -0,0 +1,27 @@
+export const COLLECTIONS_ID = 'collecties';
+export const MAPS_ID = 'mappen';
```

Comment:
map in english refers to a map with cities and roads on it
better to use `folder` for the english variable names
bonus: it will match up with the icon name :)

Reply:
fixed